### PR TITLE
[CLOUDTRUST-1649] new password generation method

### PR DESCRIPTION
**Pull request by bsoniam**

### CODE REVIEW

File: pkg/management/component.go [hunk 1]
```diff
@@ -534,7 +534,8 @@ func (c *component) ResetPassword(ctx context.Context, realmName string, userID
 	credKc.Type = &passwordType
 
 	if password.Value == nil {
-		// no password value was provided; a new password, that respects the password policy of the realm, will be generated
+		// the commented code respects the following scenario: if the realm has a pwd policy, the generated pwd respects this policy
+		/*// no password value was provided; a new password, that respects the password policy of the realm, will be generated
```
Comment (fperot74) [inline pkg/management/component.go]: /*// is a little bit weird...
It might be more readable with a new line after /*

File: Gopkg.toml [hunk 2]
```diff
@@ -27,11 +27,11 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/common-service"
-  version = "v1.0-rc6"
+  version = "=v1.0-rc6"
```
Comment (fperot74) [inline Gopkg.toml]: Why = ?
Is it an error or a feature I don't know ?

Comment (harture) [reply Gopkg.toml]: = in toml means exactly this version. If there is no =, it only means >=.
Here, I think we should remove the =

Comment (bsoniam) [reply Gopkg.toml]: I added the = as the error handling changes was causing otherwise problems;

File: pkg/management/component.go (hunk 1 above)
Comment (bsoniam) [reply pkg/management/component.go]: DONE

File: Gopkg.toml (hunk 2 above)
Comment (bsoniam) [reply Gopkg.toml]: DONE

**Pull request merged**